src/Classes/StringUtil.php

```diff
@@ -90,4 +90,41 @@ public static function formatKeywords($keywords, $charsRequired = 0)
 
         return array_values($arrKeywords);
     }
+
+    /**
+     * Convert a number of bits to a readable filesize format
+     *
+     * @param int $filesize
+     *
+     * @return string
+     */
+    public static function formatFileSize($bytes)
```

Comment:
This parameter should be an `int` according to our PHPDoc & the way we use it in the function

Reply:
The function's return type should be written in the code, not only in our PHPDoc